protocol/messenger_communities.go

```diff
@@ -67,6 +67,13 @@ const (
 	maxChunkSizeBytes    = 1500000
 )
 
+type CommunityInfoRequest struct {
+	CommunityID     string        `json:"communityId"`
+	Shard           *common.Shard `json:"shard"`
+	UseDatabase     bool          `json:"useDatabase"`
```

Comment:
```suggestion
	TryDatabase     bool          `json:"tryDatabase"`
```

It will make the intention clearer imo. It conveys the behavior of first consulting the local database before potentially reaching out to the mailserver.